busy_beaver/blueprints/slack/slash_command.py

```diff
@@ -55,8 +59,38 @@ def parse_command_text(self, command_text: str) -> Command:
         return Command(command_parts[0].lower(), args=command_parts[1:])
 
 
+#########################
+# Upcoming Event Schedule
+#########################
+@slash_command_dispatcher.on("next")
+def next_event(**data):
+    attachment = generate_next_event_message(MEETUP_GROUP_NAME)
+    return make_slack_response(attachments=attachment)
```

Comment:
Removed logic out of this view and put it into the feature specific workflow. Adding another layer of abstraction to keep this function as simple as possible.

Create a ticket to refactor the GitHub Summary Feature slash commands to follow this new pattern.